File: Back_End/src/main/java/lk/ijse/spring/entity/Customer.java
package lk.ijse.spring.entity;/*
    @author Dasun
*/

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import javax.persistence.CascadeType;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import java.util.ArrayList;
import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
@Entity
public class Customer {

    @Id
    private String customerId;
    private String name;
    private String address;
    private int contactNo;
    private String email;
    private String nicNo;
    private String licenceNo;
    private String userName;
    private String password;
    private String nicFrontImage;
    private String nicBackImage;
    private String licenceImage;

    //private String status;


    @OneToMany(mappedBy = "customer",cascade = CascadeType.ALL)
    private List<Rent> rents = new ArrayList<>();
}

File: Front_End/assets/controller/customer.js
loadCustomerDetails=()=>{

    $.ajax({
        url:baseUrl+"customer",
        method:"get",
        success:function (resp) {
            for (const i of resp.data) {

                var cusId = i.customer;
                var address = i.address;
                var contactNo = i.contactNo;
                var email = i.email;
                var licenceNo = i.licenceNo;
                var name = i.attendName;
                var nicNo = i.nicNUmber;
                var password = i.password;
                var userName = i.username;

                console.log(resp.data);

                var row = `<tr><td>${cusId}</td><td>${address}</td><td>${contactNo}</td><td>${email}</td><td>${licenceNo}</td><td>${name}</td><td>${nicNo}</td><td>${password}</td><td>${userName}</td></tr>`;
                $("#customerDetailsTableBody").append(row);

            }
        }
    });
}
